### PR TITLE
feat : added label filter

### PR DESCRIPTION
**Pull request by narharikale**
- added labels filter route

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
Looks Good

File: src/frontend/screens/Label/Label.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { useNotes } from "../../context";
+import "../notes/Notes.css";
+import "./Label.css";
+
+function Label() {
+  const { notesData } = useNotes();
+
+  const newTags = notesData.reduce((acc, curr) => {
+    return [...acc, ...curr.tags];
+  }, []);
```
Comment (t007rushi) [inline src/frontend/screens/Label/Label.jsx:12]: nice use of reduce fun

**Review by t007rushi — APPROVED** [review]
Good to merge

**Review by vaishnavipaliya11 — APPROVED** [review]
Small code changes make easier to review the code.
Code is structured properly.
Mostly you followed the good practices.

**Pull request merged**